### ISSUE TITLE
Regression: Flamegraph not visible on JFR-ingested data since 0.15.0

### ISSUE THREAD

**Issue by charlesverdad** [pyroscope-io/pyroscope#1046]
I've been using the JFR ingestion feature and noticed that the visualisation stopped working since 0.15.0

## pyroscope:0.14.0

The screenshot below shows the JFR file ingested and shown in Pyroscope 0.14.0.

![Screen Shot 2022-04-21 at 5 14 33 pm](https://user-images.githubusercontent.com/382186/164396616-4870fbb6-b11e-4b70-9b29-523b08d9e14d.png)

## pyroscope:0.15.0

The screenshot below shows the same JFR file ingested to pyroscope 0.15.0 (and 0.15.1) the flamegraph and table does not appear.  Notice in the timeline view you can see a yellow line which indicates some data has been sampled..

![Screen Shot 2022-04-21 at 5 16 01 pm](https://user-images.githubusercontent.com/382186/164396807-5ebfc636-2f7d-4beb-9a39-8254b95e3b9f.png)

The logs show that it was properly ingested with some samples

```
pyroscope_1  | time="2022-04-21T07:18:35.586620" level=debug msg=storage.Put file=" storage/storage_put.go:50" aggregationType=sum endTime="2022-04-21 07:18:35.575088595 +0000 UTC m=+206.833498303" key="test_rpc.itimer{}" samples=108 startTime="2022-04-21 07:18:35.575088479 +0000 UTC m=+206.833498202" units=samples
pyroscope_1  | time="2022-04-21T07:18:35.587079" level=debug msg=storage.Put file=" storage/storage_put.go:50" aggregationType=sum endTime="2022-04-21 07:18:35.575088595 +0000 UTC m=+206.833498303" key="test_rpc.alloc_in_new_tlab_objects{}" samples=145 startTime="2022-04-21 07:18:35.575088479 +0000 UTC m=+206.833498202" units=objects
pyroscope_1  | time="2022-04-21T07:18:35.587293" level=debug msg=storage.Put file=" storage/storage_put.go:50" aggregationType=sum endTime="2022-04-21 07:18:35.575088595 +0000 UTC m=+206.833498303" key="test_rpc.alloc_in_new_tlab_bytes{}" samples=145 startTime="2022-04-21 07:18:35.575088479 +0000 UTC m=+206.833498202" units=bytes
pyroscope_1  | time="2022-04-21T07:18:35.587616" level=debug msg=storage.Put file=" storage/storage_put.go:50" aggregationType=sum endTime="2022-04-21 07:18:35.575088595 +0000 UTC m=+206.833498303" key="test_rpc.alloc_outside_tlab_objects{}" samples=145 startTime="2022-04-21 07:18:35.575088479 +0000 UTC m=+206.833498202" units=objects
pyroscope_1  | time="2022-04-21T07:18:35.588023" level=debug msg=storage.Put file=" storage/storage_put.go:50" aggregationType=sum endTime="2022-04-21 07:18:35.575088595 +0000 UTC m=+206.833498303" key="test_rpc.alloc_outside_tlab_bytes{}" samples=145 startTime="2022-04-21 07:18:35.575088479 +0000 UTC m=+206.833498202" units=bytes
pyroscope_1  | time="2022-04-21T07:18:35.588364" level=debug msg=storage.Put file=" storage/storage_put.go:50" aggregationType=sum endTime="2022-04-21 07:18:35.575088595 +0000 UTC m=+206.833498303" key="test_rpc.lock_samples{}" samples=6 startTime="2022-04-21 07:18:35.575088479 +0000 UTC m=+206.833498202" units=samples
pyroscope_1  | time="2022-04-21T07:18:35.588629" level=debug msg=storage.Put file=" storage/storage_put.go:50" aggregationType=sum endTime="2022-04-21 07:18:35.575088595 +0000 UTC m=+206.833498303" key="test_rpc.lock_nanoseconds{}" samples=17362270 startTime="2022-04-21 07:18:35.575088479 +0000 UTC m=+206.833498202" units=nanoseconds
```

**Comment by eh-am** [comment]
This is interesting, I wasn't able to reproduce it. I added a simple example here https://github.com/pyroscope-io/pyroscope/pull/1050 if anybody wants to try.


@charlesverdad can you open the network tab and see what the `/render` call returns? Just so we have an idea where the problem lies (frontend/backend)

![image](https://user-images.githubusercontent.com/6951209/164466877-90c5e0d6-d9f1-479a-b132-75d2542dbf4c.png)

**Comment by petethepig** [comment]
Hey @charlesverdad, just a little update, we found the bug — will fix it and do a release soon.

**Comment by petethepig** [comment]
@charlesverdad This is fixed in [0.15.2](https://github.com/pyroscope-io/pyroscope/releases/tag/v0.15.2)
